_posts/2020-05-28-on-reading-comprehension-original-thoughts.md

```diff
@@ -0,0 +1,224 @@
+
+The next step of this phase is to create a heading structure for the exported highlights. I recommend to discard the author’s headings and begin anew by keeping the original chapter structure and by splitting the highlights apart based on topics. By doing so, we are able to partition information into coherent groups and provide headings which are meaningful. Apart from that, it also forces us to read through the content, which helps memorization and might unveil again some hidden criticism, question or knowledge.
+
+When all this work is done, we not only produced a multilayered representation of the knowledge of the book, but hopefully also developed a thorough understanding of the topic at hand.
+
+### Stage 5: Working through open questions and criticism
+Over time some of our questions and criticisms will come up in our thought process more often than others. When that is the case, we start new investigations, which can range from reading additional books and papers, or do some analysis, or simply to think about the problem longer.
+
+Often it is here that new ideas arise. Maybe there isn’t a ready answer to a question and we have to find it on our own. Maybe some of the criticism we have leads to a change of the underlying knowledge. It is also here, that we as readers are active and continue the story. We left the known path given by the author, his guidance and have to find our own way.
+
+## On reading original works and derivatives
+> There are, of course, many books worth reading well. There is a much larger number that should be only inspected. (*How to read a book*, Mortimer Adler)  
+
+So far we discussed how to accomplish the goals of *Comprehension*, *Recall* and having *Original Thoughts* set out at the beginning. In this chapter we will examine which types of written works are best suited for this set of goals. Depending on the expected level of understanding we can make a distinction between what can be called *Foundational Works* and *Derivatives*.
+
+### Foundational Works
+It is the first written work which puts a concept or idea in a coherent form. It is often the foundation of a field of knowledge. Examples are [The Wealth of Nations](https://en.wikipedia.org/wiki/The_Wealth_of_Nations) by Adam Smith which is the foundational work for classical economics, or  Albert Einstein’s Special and General Theory of Relativity which created the basis of our modern understanding of gravity.
+
+### Derivatives
+All works which are based on that foundations. They extend them, provide interpretations and opinions, or produce a partial, often condensed view of the matter. Translations can also fall into this category, but we will discuss this specific topic later on.
```

Comment:
```suggestion
All works which are based on that foundation. They extend them, provide interpretations and opinions, or produce a partial, often condensed view of the matter. Translations can also fall into this category, but we will discuss this specific topic later on.
```

*All works which are based on that foundation* is this meant to define *Derivatives*? It's a sentence fragment and should be made whole.

Reply:
Maybe _Works which are based on or derived from these foundations are Derivatives._ ?